clients/fides-js/package.json

```diff
@@ -70,8 +70,9 @@
     "rollup-plugin-postcss": "^4.0.2",
     "rollup-plugin-visualizer": "^5.12.0",
     "ts-jest": "^29.1.0",
+    "typedoc": "^0.25.13",
```

Comment:
I should have included this previously! It was getting pulled in as a peer dependency.